components/src/dynamo/trtllm/workers/llm_worker.py

```
@@ -291,6 +297,20 @@ async def init_llm_worker(
         # TODO: Support extra engine args from json file as well.
+        # Parse YAML once to (a) capture user-supplied return_perf_metrics
+        # for later re-application, and (b) detect collisions with _warn_override.
+        _parsed_extra = {}
+        try:
+            import yaml
+
+            _parsed_extra = yaml.safe_load(config.extra_engine_args) or {}
+            if not isinstance(_parsed_extra, dict):
+                _parsed_extra = {}
+        except Exception:
+            pass  # YAML parse errors are handled by update_llm_args_with_extra_options
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)